avm/src/vm/program.rs

```diff
@@ -84,6 +84,11 @@ impl Program {
     let httpconn = i64::from(BuiltInEvents::HTTPCONN);
     self.event_pls.insert(httpconn, 0);
     self.event_handlers.insert(httpconn, Vec::new());
+    // NOP
+    // the compiler should prevent programs from handling this event.
```

Comment:
but sometimes it does? this comment is confusing to me, can you elaborate within it?

Reply:
i just mean that there shouldn't be a way to say `on NOP { ... }`

Reply:
ooooh gotcha. maybe `the compiler does not allow explicitly defining handlers for this event and it's used internally by the compiler for closures in unused conditional branches`?